packages/utils/interactive-hooks/src/useViewCommandFocus.macos.ts

```diff
@@ -22,19 +22,15 @@ export function useViewCommandFocus(
 
   const _setNativeRef = setAndForwardRef({
     getForwardedRef: () => forwardedRef,
-    setLocalRef: localRef => {
+    setLocalRef: (localRef) => {
       focusRef.current = localRef;
 
       /**
        * Add focus() as a callable function to the forwarded reference.
        */
       if (localRef) {
         localRef.focus = () => {
-          NativeModules.UIManager.dispatchViewManagerCommand(
-            findNodeHandle(localRef),
-            NativeModules.UIManager.getViewManagerConfig('RCTView').Commands.focus,
-            null,
-          );
+          UIManager.dispatchViewManagerCommand(findNodeHandle(localRef), UIManager.getViewManagerConfig('RCTView').Commands.focus, null);
```

Comment:
Dumb question I've had that I haven't been able to answer elsewhere:
This `UseViewCommandFocus` hook seems redundant, since the standard ref from React should have a `.focus()` method on it. But if I remove the hook altogether, it seems imperative focus stops working on win32. Would you happen to know why?

Reply:
Sounds like a bug.  We should make sure that view.focus() does the right thing in win32.